### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by TheYahya** [sherlock-project/api#1]
It's sherlock api repo, and this issue is for any thoughts and recommendations.
My idea is to develop an API and then creating web app (react.js) and mobile app with the api, And for api to keep it simple and practical: 
- Using [sherlock](https://github.com/sherlock-project/sherlock) as a pypi module
- Using [DRF](https://github.com/encode/django-rest-framework)

**Comment by mchahed99** [comment]
Awesome 👍

**Comment by rajgoesout** [comment]
django or flask?

**Comment by jcs090218** [comment]
Cool! To bad I'm not really good at the web development. Working on it now! :(

**Comment by TheYahya** [comment]
I prefer django because of faster bootstarping the project, and having more structure that in a open source project all should follow. @rajdeepbharati

**Comment by rajgoesout** [comment]
We can use DRF to quickly build a prototype, since this won't have too many endpoints @TheYahya ?

**Comment by TheYahya** [comment]
Yes, It seems cool framework for developing api ☺️ , good idea 👍 @rajdeepbharati

**Comment by theapache64** [comment]
Any updates on this ?

**Comment by TheYahya** [comment]
@theapache64
I set some structure and write some codes in django. But before that [pypi](https://github.com/sherlock-project/sherlock/pull/112) modulation should be completed

**Comment by rajgoesout** [comment]
@TheYahya I could write a POC of the frontend using mock data in Vue.js

**Comment by TheYahya** [comment]
Hi @rajdeepbharati, 
We want to write the front-end app with react.js, If you interested to work on that, we could talk the details and structure of the app.

**Comment by rajgoesout** [comment]
Sure, I would be happy to do it in React

**Comment by TheYahya** [comment]
Cool, 
I just send you an email.

**Comment by chrisvxd** [comment]
Hey team! 👋

I wanted to reach out and see if I could help you build an API on top of Sherlock?

For context, I'm exploring new ways to fund OSS with @transitive-bullshit, and [our current approach](https://github.com/saasify-sh/saasify) is to make it really easy to build rate-limited APIs on top of existing projects.

We've [built quite a few](https://saasify.sh/#/showcase) at this point, but are looking to build more.

I figured it would save you a lot of work. We'd be happy to handle dev, auth, billing and support so you wouldn't have to do anything. You'd be able to take ownership of the boilerplate code we write, but it's pretty trivial and all controlled via [a CLI we built](https://github.com/saasify-sh/saasify/tree/master/packages/saasify-cli).

Looking forward to hearing your thoughts! 🙏

**Comment by jcs090218** [comment]
@TheYahya Any progress on this? I just started learning DRF and maybe I can start with the basic setup. Can you tell me the structure that you are seeking to build?

**Comment by TheYahya** [comment]
Hi dear @jcs090218,

There haven't been any progress on this; I don't have any particular structure on my mind right now, But I suppose that it would follow best practices.
So It would be great if you could submit a PR.

Thank you.

**Comment by CustomIcon** [comment]
Hello fellow Contributors and Maintainers. I was thinking to make one with fast api but i came across this discussion.

**Comment by voidcoefficient** [comment]
Is there a list of wanted additions for new maintainers to know where to help? This project interests me

**Comment by aliqq7** [comment]
Me pls
